Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1143,4 +1154,55 @@ def problem(text)
       @problems << text
     end
   end
+
+  class TapAuditor
+    attr_reader :name, :path, :tap_audit_exceptions, :problems
+
+    def initialize(tap, options = {})
```

Comment:
Please use keyword arguments for this. Would be nice to get rid of our uses of `options = {}` now we have them.